.expeditor/verify_private.pipeline.yml

```diff
@@ -929,3 +929,16 @@ steps:
       executor:
         linux:
           privileged: true
+
+  - label: "ocid"
+    command:
+      - integration/run_test integration/tests/testing_ocid.sh
+    timeout_in_minutes: 30
```

Comment:
reduce the timeout as we won't have many testcases to run.

Reply:
Okay @RajeshPaul38 should we keep it as 20?

Reply:
I have kept it as 15 for now. Let check if it works properly for is. We will change accordingly